src/main/jbake/content/hawkular-metrics/docs/user-guide/index.adoc

```diff
@@ -1888,10 +1896,155 @@ Resulting in:
+
+The MGT could look something like this (as an hAlerting full trigger JSON):
+
+----
+{
+"trigger": {
+  "type": "GROUP",
+  "tenantId": "MyProject",
+  "id": "MGT-FreeSpace",
+  "name": "MGT FreeSpace",
+  "description": "An alert to notify admins that disk space is running low.",
+  "autoDisable": true,
+  "enabled": true,
+  "tags": {
+    "HawkularMetrics": "GroupTrigger",
+    "DataIds": "FreeSpace",
+    "FreeSpace": "name = FreeSpace",
+    "SourceBy": "*"
+  }
+},
+"conditions": [
+  {
+    "type": "THRESHOLD",
+    "dataId": "FreeSpace",
+    "operator": "LT",
+    "threshold": 10
+  }
+]
+}
+----
+
+Pay careful attention to the tagging on the trigger, it's the tags that make this a Metrics Group Trigger.
+
+* `"HawkularMetrics": "GroupTrigger"`
+** This tag identifies the group trigger as an MGT to be processed by the hMetrics alerter.
+* `"DataIds": "FreeSpace"`
+** This tag declares the dataIds used in the group trigger conditions.
+** In our example we have only one condition using only one dataId, "FreeSpace".
+* `"FreeSpace": "name = FreeSpace"`
+** This tag provides the metrics tag query that identifies the set of metrics to use for member trigger creation.
+** In this example each metric tagged with {"name": "FreeSpace"} will be used.
+** See <<Tag Filtering>> for more details.
+* `"SourceBy": "*"`
+** This tag is used to make ensure member triggers use related metrics. It is described in more detail below.
+** When using a single dataId it will commonly be set to "*" because relating multiple metrics is not applicable.
+
+==== Tagging
+To use Metrics Group Triggers there is necessary tagging to be applied not only the MGT itself but in most cases the metric definitions also require tags.  This section discusses both the trigger and metric tagging.
+
+===== Metric Group Trigger Tagging
+Each Metrics Group Trigger has several tags to guide its handling:
+
+|===
+|Tag Name |Tag Value |Notes
+
+|"HawkularMetrics"
+|"GroupTrigger"
+| Required to be processed as Metrics Group Trigger
+
+|"DataIds"
+|DataId1..DataIdN
+|The DataIds used in the trigger conditions, required
+
+|DataId1
+|tag query
+|Metrics tag query for dataId1 metrics
+
+|...
+|...
+|...
+
+|DataIdN
+|tag query
+|Metrics tag query for dataId1 metrics
+
+|"SourceBy"
+|TagName1..TagNameN
+.|see <<SourceBy>> below
+
+|===
+
+====== SourceBy
+The __SourceBy__ tag specifies the metric tag names used to determine the member trigger population. As an example, consider an MGT with one CompareCondition: `HeapUsed > 80% HeapMax`. We want to compare `HeapUsed` to `HeapMax` but of course each comparison should be done on metrics from the same machine. We don't want to compare all `HeapUsed` to all `HeapMax` for every machine in the data center. We would tag our MGT something like this:
```

Comment:
Question, it describes "metric tag names", I guess that SourceBy is expecting just a metric, or it can be more than one metric ?